harvard-lts/epadd-curator-app
# Fix question mark at end of export transfer

**Fix question mark at end of export transfer**
* * *

**Ticket Link**: [LTSEPADD-58](https://jira.huit.harvard.edu/browse/LTSEPADD-58)

# What does this Pull Request do?
This PR strips the extra newline characters when setting the OSN from drsConfig.txt so a '?' is not appended to the package_id when transferring from s3 to the dropbox. It also modifies the way parameters are passed when calling the system test.

# How should this be tested?

1. Already deployed to dev through `trial` branch
2. Call `http://ltsds-cloud-dev-1.lib.harvard.edu:10586/testExport`
3. Confirm an export named with the format of `osn_{timestamp` is present in the dropbox
4. Call `http://ltsds-cloud-dev-1.lib.harvard.edu:10586/testExport?osn={aUniqueOSN}`
5. Confirm an export named your provided OSN is present in the dropbox with no '?' appended
6. Run epadd-curator-app locally
7. Exec into the container and run unit tests with `python3 -m unittest scripts/unit_tests.py`

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? yes - no new methods
- integration tests? yes

# Interested parties
@ives1227
